### DISCUSSION TITLE
trying-out-the-open-source-ecommerce-platform-nopcommerce-using-docker/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#105] [Announcements]
# trying-out-the-open-source-ecommerce-platform-nopcommerce-using-docker/

In this post I show to run nopCommerce locally using docker-compose, show how to bootstrap the app, and take a brief tour of the app

https://andrewlock.net/trying-out-the-open-source-ecommerce-platform-nopcommerce-using-docker/

**Comment by ankushjain358** [comment]
So should we copy the files generated in AppData folder post installation inside our code repository and check-in then in source control? Otherwise how the app know about connection string if we redeploy or re-publish the app.

[a comment by an automated account is not shown]
